.goreleaser.yml

```diff
@@ -45,12 +44,47 @@ builds:
     ignore: *build-ignore
     hooks: *build-hooks
     main: ./cmd/populator
-    binary: 'populator-{{ .Os }}-{{ .Arch }}'
-    no_unique_dist_dir: true
+    binary: 'populator'
 
 archives:
-  - format: binary
-    name_template: '{{ .Binary }}{{ if .Arm }}v{{ .Arm }}{{ end }}{{ if .Mips }}_{{ .Mips }}{{ end }}'
+  - id: capact-archive
+    name_template: &archives-name-template '{{ .Binary }}-{{ .Os }}-{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}{{ if .Mips }}_{{ .Mips }}{{ end }}'
+    format: &archives-format binary
+    builds:
+    - capact
+
+  - id: populator-archive
+    name_template: *archives-name-template
+    format: *archives-format
+    builds:
+    - populator
+
+brews:
+  - name: capact
+    ids:
+    - capact-archive
+    homepage: &homebrew-homepage https://github.com/capactio/homebrew-tap
+    description: "Capact CLI is a command-line tool, which manages Capact resources."
+    license: "Apache License 2.0"
+    tap: &homebrew-tap
+      owner: capactio
+      name: homebrew-tap
+    commit_author: &homebrew-commit-author
+        name: Capact Bot
+        email: capactbot@capact.io
+    test: |
+      system "#{bin}/capact version"
+
+  - name: populator
+    ids:
+    - populator-archive
+    homepage: *homebrew-homepage
+    description: "Populator is a command-line tool, which helps to populate various Capact content."
+    license: "Apache License 2.0"
+    tap: *homebrew-tap
+    commit_author: *homebrew-commit-author
+    test: |
+      system "#{bin}/populator help"
 
```

Comment:
Consider removing gcs upload: https://github.com/capactio/capact/blob/392c85d86c9d9857daf401c059860e9ed13560c4/.goreleaser.yml#L80-L84

As binaries are already available both on GitHub and via homebrew tap.

If you will change it, also you can remove GCP auth on GitHub pipelines.

Reply:
Removed although I think we cannot remove GCP auth from the release pipeline. As the last step in the pipeline there is:
```
      - name: Release Helm charts
        run: make release-charts
```
and it use `gsutil`.